js/snippetPreview.js

```diff
@@ -397,3 +672,141 @@ YoastSEO.SnippetPreview.prototype.setFocus = function( ev ) {
 		}
 	}
 };
+
+/**
+ * Binds the reloadSnippetText function to the blur of the snippet inputs.
+ */
+SnippetPreview.prototype.bindEvents = function() {
+	var targetElement,
+		elems = [ "title", "slug", "meta-description" ],
+		focusBindings = [
+			{
+				"click": "title",
+				"focus": "title"
+			},
+			{
+				"click": "urlPath",
+				"focus": "urlPath"
+			},
+			{
+				"click": "urlBase",
+				"focus": "urlPath"
+			},
+			{
+				"click": "metaDesc",
+				"focus": "metaDesc"
+			}
+		];
+
+	for ( var i = 0; i < elems.length; i++ ) {
```

Comment:
Is there a reason why this is being done in a for-loop vs the usage of forEach as seen on: https://github.com/Yoast/YoastSEO.js/pull/211/files#diff-1639d25d23d5ae28ab474b3f1db631b1R725?